app/src/sharedTest/java/org/oppia/app/topic/review/TopicReviewFragmentTest.kt

```diff
@@ -123,6 +135,58 @@ class TopicReviewFragmentTest {
     }
   }
 
+  @Test
+  fun testTopicReviewFragment_loadFragment_configurationChange_checkTopicThumbnail_isCorrect() {
+    launchTopicActivityIntent(internalProfileId, FRACTIONS_TOPIC_ID).use {
+      onView(
+        allOf(
+          withText(TopicTab.getTabForPosition(3).name),
+          isDescendantOfA(withId(R.id.topic_tabs_container))
+        )
+      ).perform(click())
+      it.onActivity { activity ->
+        activity.requestedOrientation = Configuration.ORIENTATION_LANDSCAPE
```

Comment:
For configuration change we are using a function `orientationLandscape` present in `OrientationChangeAction` file. Use that only.

Reply:
Ditto elsewhere

Reply:
Changes done